src/amp-story-player/amp-story-player-impl.js

```diff
@@ -486,6 +487,40 @@ export class AmpStoryPlayer {
     }
   }
 
+  /**
+   * Navigates stories given a number.
+   * @param {number} storyDelta
+   * @private
+   */
+  go(storyDelta) {
+    if (this.currentIdx_ + storyDelta >= this.stories_.length || this.currentIdx_ + storyDelta < 0) {
+      throw new Error('Out of Story range.')
+    }
+    
+    if (storyDelta === 0) {
+      return;
+    }
+
+    const newIndex = this.stories_[this.currentIdx_ + storyDelta];
```

Comment:
`this.stories_` is an array containing anchor elements that represent a story (`<a href="story.html">`), so `newIndex` here is not actually an index but an element. So the line below will not be doing what you want it to do.